Transfer/Sources/ListTransferDestinationCell.swift

```diff
@@ -0,0 +1,106 @@
+// sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or
+// substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+#if !COCOAPODS
+import Common
+#endif
+import HyperwalletSDK
+import UIKit
+
+/// ListTransferDestinationCell
+final class ListTransferDestinationCell: UITableViewCell {
+    static let reuseIdentifier = "listTransferDestinationCellIdentifier"
+
+    // MARK: Life cycle
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: .subtitle, reuseIdentifier: reuseIdentifier)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    override public func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+        imageView?.backgroundColor = Theme.Icon.primaryBackgroundColor
+    }
+
+    override public func setHighlighted(_ highlighted: Bool, animated: Bool) {
+        super.setHighlighted(highlighted, animated: animated)
+        imageView?.backgroundColor = Theme.Icon.primaryBackgroundColor
+    }
+
+    // MARK: Theme manager's proxy properties
+    @objc dynamic var titleLabelFont: UIFont! {
+        get { return textLabel?.font }
+        set { textLabel?.font = newValue }
+    }
+
+    @objc dynamic var titleLabelColor: UIColor! {
+        get { return textLabel?.textColor }
+        set { textLabel?.textColor = newValue }
+    }
+
+    @objc dynamic var subTitleLabelFont: UIFont! {
+        get { return detailTextLabel?.font }
+        set { detailTextLabel?.font = newValue
+            detailTextLabel?.font = newValue }
+    }
+}
+
+extension ListTransferDestinationCell {
+    /// Fill `ListTransferDestinationCell` related fields
+    ///
+    /// - Parameter transferMethod: a transfer method which contains the info needs to be filled to the cell.
+    func configure(transferMethod: HyperwalletTransferMethod) {
+        textLabel?.accessibilityIdentifier = "transferDestinationTitleLabel"
+        textLabel?.text = transferMethod.type?.lowercased().localized()
+        textLabel?.adjustsFontForContentSizeCategory = true
+        textLabel?.numberOfLines = 0
+        textLabel?.lineBreakMode = .byWordWrapping
+
+        detailTextLabel?.accessibilityIdentifier = "transferDestinationSubtitleLabel"
```

Comment:
transferDestinationSubtitleLabel -> transferDestinationDetailTextLabel

Reply:
tests were failing if not using `transferDestinationTitleLabel` and `transferDestinationSubtitleLabel`